### ISSUE TITLE
Elevated circulating methylsuccinic acid concentration

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6020]
**Preferred term label:** Elevated circulating methylsuccinic acid concentration

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of methylsuccinic acid in the blood is above the normal range., PMID: 20280760

**Parent term (use hpo.jax.org/app)** Dicarboxylic acidemia HP:0040145

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
This is the serum finding related to https://hpo.jax.org/app/browse/term/HP:0410153 in urine.
Disease annotations include: OMIM:618156

**Comment by pnrobinson** [comment]
addded as HP_0033146
